### PR TITLE
use Go standard errors

### PR DESCRIPTION
**Pull request by mmorel-35**
* Since Dec 1, 2021, github.com/pkg/errors is an archived project.
* Since go 1.13, there is a standard errors and fmt modules that provides equivalent ways to handle errors.

Signed-off-by: Matthieu MOREL <matthieu.morel35@gmail.com>

### CODE REVIEW

**Review by metalmatze — APPROVED** [review]

**Comment by mmorel-35** [comment]
Anything left to merge this ?

File: tools/fake-webserver/server.go [hunk 1]
```diff
@@ -88,49 +88,49 @@ type responseOpts struct {
 }
 
 var opts = map[string]map[string]responseOpts{
-	"/api/foo": map[string]responseOpts{
-		"GET": responseOpts{
+	"/api/foo": {
+		"GET": {
```
Comment (bboreham) [inline tools/fake-webserver/server.go]: This change doesn't seem covered by the PR or commit description.

**Review by bboreham — COMMENTED** [review]
Looks plausible to me, but I am not familiar with the code.

It seems some more linting was turned on, which prompted some other, mostly cosmetic, changes.  I always look for consistency between the description and the diffs, since it can reveal changes included by mistake.

File: tools/fake-webserver/server.go (hunk 1 above)
Comment (mmorel-35) [reply tools/fake-webserver/server.go]: That's gofumpt simplifying code.

**Comment by mmorel-35** [comment]
I'll narrow this to fit title and provide another PR for the formarring

**Comment by bwplotka** [comment]
Friendly ping, would be nice to improve things (: Thanks!

**Comment by mmorel-35** [comment]
@bwplotka ,
That's ready for review

File: tools/fake-webserver/server.go (hunk 1 above)
Comment (mmorel-35) [reply tools/fake-webserver/server.go]: I did a rollback on formatting linters

**Review by bwplotka — APPROVED** [review]
Thanks!!!

**Pull request merged**